src/stock-adjustment-mobile/adjustment-form.component.jsx

```diff
@@ -97,12 +97,24 @@ const AdjustmentForm = ({ stockAdjustmentCreationService,
             indexOfProductToEdit: index
         };
         localStorage.setItem('stateLocation', JSON.stringify(stateLocation));
+        removeToasts();
         history.push({
             pathname: `/make${adjustmentType}AddProducts/editProduct${adjustmentType}`,
             state: stateLocation
         });
     };
 
+    const removeToasts = () => {
+        let listToRemove = toastList;
+        if (listToRemove.length) {
+            listToRemove = deleteToast(listToRemove[0].id, listToRemove);
+            toastList = listToRemove;
```

Comment:
Are you sure you need this temporary variable? Does
```
if (toastList.length) {
  toastList = deleteToast(toastList[0].id, toastList);
```
not work?

Reply:
Need to check, but you are right, your proposal should work too and will simplify this code.

Reply:
I've just checked, that proposal doesn't work.